include/zephyr/drivers/modem/murata-1sc.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2022 T-Mobile USA, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <zephyr/net/buf.h>
+
+#define MDM_IMEI_LENGTH			  16
+#define MDM_IMSI_LENGTH			  16
+#define MDM_ICCID_LENGTH		  32
+#define MDM_PHN_LENGTH                    16
+#define TLS_MURATA_USE_PROFILE            99
+#define TLS_MURATA_CLIENT_VERIFY          100
+#define MAX_IP_RESP_SIZE                  256
+#define MDM_GOLDEN_LEN			  16
+#define MAX_PSM_RESP_SIZE                 128
+#define MAX_EDRX_RESP_SIZE                128
```

Comment:
I wonder if all these defines should have a `MDM_1SC` prefix...

Reply:
+1